Library/Homebrew/extend/ARGV.rb

```diff
@@ -59,8 +59,8 @@ def kegs
 
       raise NoSuchKegError, rack.basename if dirs.empty?
 
-      linked_keg_ref = HOMEBREW_LIBRARY.join("LinkedKegs", rack.basename)
-      opt_prefix = HOMEBREW_PREFIX.join("opt", rack.basename)
+      linked_keg_ref = HOMEBREW_LINKED_KEGS/rack.basename
+      opt_prefix = HOMEBREW_PREFIX/"opt/#{rack.basename}"
```

Comment:
Given it's a single path below `HOMEBREW_PREFIX` I think it's fine for now. `LinkedKegs` is a bit more involved.